easybuild/easyblocks/generic/fortranpythonpackage.py

```diff
@@ -51,6 +51,10 @@ def build_step(self):
             self.cfg.update('buildopts', "--compiler=intel --fcompiler=intelem")
             cmd = "%s %s setup.py build %s" % (self.cfg['prebuildopts'], self.python_cmd, self.cfg['buildopts'])
 
+        elif comp_fam == toolchain.FUJITSU:   # @UndefinedVariable
+            self.cfg.update('buildopts', "--fcompiler=fj")
```

Comment:
this of course only works if the package distutils (e.g. numpy's) has been patched with an `fj` compiler, so we should check this here, or even patch distutils here?

Reply:
Do you have an example of the required patch?

Reply:
https://gist.github.com/migueldiascosta/ffaa2061a95ee13736b4e53f3355995e

Reply:
the fujitsu compiler definition has been added to numpy in https://github.com/numpy/numpy/pull/17792